Aikido.Zen.Test/testdata/data.PathTraversalDetector.json

```diff
@@ -178,5 +178,42 @@
     "path": "C:\\files\\\\\\.\\test.txt",
     "description": "Windows device namespace syntax (backslash)",
     "isTraversal": false
+  },
+  {
+    "input": "%PROGRAMFILES%",
+    "path": "%PROGRAMFILES%\\app\\config.ini",
+    "description": "Program Files environment variable",
+    "isTraversal": true
+  },
+  {
+    "input": "%USERPROFILE%\\Documents\\file.txt",
+    "path": "%USERPROFILE%\\Documents\\file.txt",
+    "description": "Windows environment variable path",
+    "isTraversal": true
+  },
+  {
+    "input": "/etc/passwd",
+    "path": "/etc/passwd",
+    "description": "Absolute Linux path",
+    "isTraversal": true
+  },
+  {
+    "input": "/home/user/file.txt",
+    "path": "/home/user/file.txt",
```

Comment:
can we add some tests with absolute paths but where the end is not the same or where the end in user input is different from what is already in path (would then not be path trav)